src/site/stages/build/drupal/menus.js

```diff
@@ -264,6 +264,12 @@ function makeSection(hostUrl, hub, arrayDepth, promo, pages) {
  * @return {Array} headerData - Menu information formatted for the megaMenu React widget.
  */
 function formatHeaderData(buildOptions, contentData) {
+  if (!contentData?.data) {
+    // eslint-disable-next-line no-console
+    console.warn('formatHeaderData has no data');
+    return null;
+  }
+
```

Comment:
I have no idea why this is happening, nor what the impact of returning `null` here is.